src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -916,8 +951,9 @@ class MacroAssembler: public Assembler {
   void revbw(Register Rd, Register Rs, Register tmp1 = t0, Register tmp2= t1);  // reverse bytes in lower word, sign-extend
   void revb(Register Rd, Register Rs, Register tmp1 = t0, Register tmp2 = t1);  // reverse bytes in doubleword
 
+  void ror_reg(Register dst, Register src, Register shift, Register tmp = t0);
```

Comment:
Hi, Could you please merge master? Then we can rename this routine to:
`void ror(Register dst, Register src, Register shift, Register tmp = t0);`

Reply:
> Hi, Could you please merge master? Then we can rename this routine to: `void ror(Register dst, Register src, Register shift, Register tmp = t0);`

Fixed.